birdclef/experiment/model.py

```diff
@@ -120,3 +120,49 @@ def __init__(
             nn.ReLU(inplace=True),
             nn.Linear(hidden_layer_size, num_labels),
         )
+        
+
+class LSTMClassifier(LinearClassifier):
+    # TODO: in progress
+    def __init__(
+        self,
+        num_features: int,
+        num_classes: int,
```

Comment:
this is likely going to cause some issues with deserialization, see https://github.com/dsgt-kaggle-clef/birdclef-2024/pull/31 for the solution to fix this.